content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -32,43 +32,51 @@ Or, if there are already existing programs, select an existing program to edit i
 
-The program form contains the following options:
+.. image:: loyalty_discount/price-programs.png
+   :align: center
+   :alt: Program options on the loyalty program form.
 
-- :guilabel:`Program Name`: Enter the name of the program. The program name is *not* visible to the
+The program form contains the following fields:
+
+- :guilabel:`Program Name`: enter the name of the program. The program name is *not* visible to the
   customer.
-- :guilabel:`Program Type`: Select the desired :ref:`program type
+- :guilabel:`Program Type`: select the desired :ref:`program type
   <sales/pricing_management/program-types>`.
-- :guilabel:`Currency`: Select the currency used for the program.
-- :guilabel:`Points Unit`: Enter the name of the points used for the :guilabel:`Loyalty Cards`
+- :guilabel:`Currency`: select the currency used for the program.
+- :guilabel:`Pricelist`: if desired, select a pricelist from the drop-down menu to have this loyalty
+  program applied to a specific pricelist (and customers attached to the pricelist). More than one
+  pricelist can be selected in this field. When a single loyalty program is linked to several
+  pricelists, it makes it viable for different customer segments to have different pricelists, but
+  the *same* loyalty programs. If this field is left blank, the program applies to everyone,
+  regardless of pricelist.
+- :guilabel:`Points Unit`: enter the name of the points used for the :guilabel:`Loyalty Cards`
   program (e.g. `Loyalty Points`). The points unit name *is* visible to the customer. This field is
   only available when the :guilabel:`Program Type` is set to :guilabel:`Loyalty Cards`.
-- :guilabel:`Validity`: Select the date until which the program is valid. Leave this field blank for
-  no end date, meaning the program is always valid and does not expire.
-- :guilabel:`Limit Usage`: Check this box and enter a number to limit the number of times the
-  program can be used during the :guilabel:`Validity` period.
-- :guilabel:`Company`: In the case of multiple companies, choose the company for which the program
+- :guilabel:`Start Date`: select the date on which the program becomes valid. Leave this field blank
+  if the program should always be valid and not expire.
+- :guilabel:`End Date`: select the date on which the program stops being valid. Leave this field
+  blank if the program should always be valid and not expire.
+- :guilabel:`Limit Usage`: if desired, tick this checkbox, and enter a number of :guilabel:`usages`
+  to limit the number of times the program can be used during the validity period.
+- :guilabel:`Company`: in the case of multiple companies, choose the company for which the program
   is available.
-- :guilabel:`Available On`: Select the app(s) on which the program is available.
-- :guilabel:`Website`: Select the website(s) on which the program is available. Leave this field
+- :guilabel:`Available On`: select the app(s) on which the program is available.
```

Comment:
```suggestion
- :guilabel:`Available On`: select the apps on which the program is available.
```

According to the Linter/style guide, I believe we are supposed to default to the plural.